src/htcdaskgateway/cluster.py

```diff
@@ -25,15 +25,27 @@ def __init__(self, **kwargs):
         self.batchWorkerJobs = []
         self.defaultImage = 'coffeateam/coffea-base-almalinux8:0.7.22-py3.10'
         self.cluster_options = kwargs.get('cluster_options')
+        if 'image_registry' in kwargs:
+            self.image_prefix = kwargs.pop('image_registry')
+        else:
+            self.image_prefix = 'registry.hub.docker.com'
         
         #set default image if the image is not specified by user
         if not kwargs.get('image') and (not self.cluster_options or not self.cluster_options.image):
             kwargs['image'] = self.defaultImage
             print("Selected Image: ", kwargs['image'])
             self.condor_image = self.defaultImage
         else:
+            print("Selected Image: ", kwargs['image'])
             self.condor_image = kwargs.get('image')
```

Comment:
Maybe we can rename this to 'apptainer_image'?

Reply:
Maybe? I think it is just so people know what image the workers have, as a user I think either works